policyengine_us/parameters/gov/states/oh/tax/income/credits/adoption/amount/min.yaml

```
@@ -0,0 +1,18 @@
+description: Ohio issues the adoption credit of this minumum amount.
+values:
+  2021-01-01: 1_500
+  2023-01-01: 0
+metadata:
+  unit: currency-USD
+  period: year
+  label: Ohio adoption credit minimum amount
+  reference:
+    - title: Ohio 2022 Instructions for Filing Original and Amended - Line 18 – Ohio Adoption Credit
+      href: https://tax.ohio.gov/static/forms/ohio_individual/individual/2022/it1040-sd100-instruction-booklet.pdf#page=21
+    - title: Ohio Income - Individual Credits (Education, Displaced Workers & Adoption) 15. - 20.
+      href: https://tax.ohio.gov/wps/portal/gov/tax/help-center/faqs/income+-+individual+credits/income-individual-credits
+    # The ohio adoption credit has been repealed in the newest revised code, but can still be found in the previous legal code and 2022 tax form
```

```suggestion
    # The Ohio adoption credit has been repealed in the newest revised code, but can still be found in the previous legal code and 2022 tax form
```
Also cap comment lines at 79 characters